src/alfasim_sdk/alfasim_sdk_api/api.h

```diff
@@ -476,6 +476,34 @@ DLL_EXPORT int get_flow_pattern(
     int* size
 );
 
+/*!
+    Gets the liquid-liquid flow pattern for each control volume. Related to the UCM
+    liquid-liquid system.
+
+    List of possible values of `Liquid-Liquid Flow Pattern` is:
+    - `0 - Unknown`: Unknown Flow Pattern.
+    - `1 - Ambivalent`: Ambivalent Flow Pattern between Dispersed Oil and Dispersed Water.
```

Comment:
It is better to be consistent with the capital letter. We have "Dispersed Oil", "Dispersed oil", "continuous oil", "continuous Oil". All prefer all letters in lower case